modules/s3-remote-state/README.md

```diff
@@ -25,8 +27,16 @@ variable "principals" {
 
 provider "aws" { }
 
+#terraform {
+#  backend "s3" {
+#    encrypt = "true"
+#    bucket  = "fpco-tf-remote-state-test"
+#    key     = "mod/s3-remote/terraform.tfstate"
```

Comment:
I'd recommend a `key` set to `remote-state/terraform.tfstate`, or `remote-state-storage`, or `remote-state-store`, or similar.

Reply:
fixed